c7n/resources/s3.py

```diff
@@ -3469,3 +3469,37 @@ def process_bucket(self, b):
                 raise
             controls = {}
         b[self.annotation_key] = controls.get('OwnershipControls')
+
+
+@filters.register('bucket-acl')
```

Comment:
Make a comment so that we can easily identify our S1 custom filter. ex:                                                                                                 
 # ############### S1 New Filters- start ##################